experimental/group/group.go

```diff
@@ -0,0 +1,50 @@
+package group
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/inngest/inngestgo/step"
+)
+
+type Result struct {
+	Error error
+	Value any
+}
+
+func Parallel(
+	ctx context.Context,
+	fns ...func(ctx context.Context,
+	) (any, error)) []Result {
+	ctx = context.WithValue(ctx, step.ParallelKey, true)
+
+	results := []Result{}
+	isPlanned := false
+	ch := make(chan struct{}, 1)
+	for _, fn := range fns {
+		fn := fn
+		go func(fn func(ctx context.Context) (any, error)) {
+			defer func() {
+				if r := recover(); r != nil {
+					if _, ok := r.(step.ControlHijack); ok {
+						isPlanned = true
+					} else {
+						// TODO: What to do here?
+						fmt.Println("TODO")
```

Comment:
Should we just repanic for now? I suppose this might be where we capture the non-Inngest panic and could return it as an error to Inngest in the future?

Reply:
Do you think we should recover the panic and repanic it outside the goroutine? I _think_ that'll let our normal non-control-flow panic recovery logic work

Reply:
Ooh yeah if we already have something that captures panics then that'd be awesome.